script-engine/src/main/java/org/ihtsdo/termserver/scripting/reports/qi/TemplateList.java

```diff
@@ -34,7 +36,7 @@ public void init (JobRun jobRun) throws TermServerScriptException {
 	
 	public void postInit() throws TermServerScriptException {
 		String[] columnHeadings = new String[] {"Template Domain, Template Name / QI Path, Source, Documentation", 
-												""};
```

Comment:
This report would fail because it was trying to add data to a sheet with no column headers; I have therefore added Template Name and Invalid Reason.